Sources/SwiftDocC/Infrastructure/DocumentationBundle.swift

```diff
@@ -23,8 +23,6 @@ import Foundation
  - ``displayName``
  - ``identifier``
  - ``version``
- - ``defaultCodeListingLanguage``
- - ``defaultAvailability``
```

Comment:
Why removing these symbols from here? There are also part of the Documentation Bundle.

Reply:
They are not. These links don't resolve to anything.